src/js/subtraction/components/Attribution.tsx

```diff
@@ -11,7 +16,19 @@ const StyledNoneFoundAttribution = styled.div`
     top: 50%;
 `;
 
-export function SubtractionAttribution({ handle, time }) {
+type SubtractionAttributionProps = {
+    /* The user handle */
+    handle: string;
+    /* The time of the subtraction */
+    time?: string;
+};
+
+/**
+ * Subtraction attribution with user handle and time.
+ * If only user is provided, renders without time.
+ * If neither user nor time is provided, renders default message.
```

Comment:
Details can be read from the code, we want to keep the descriptions succinct:

```suggestion
 * Formatted attribution showing creating user's handle and time of creation